Polkadot nominator and validator guides (#467)
w3f/polkadot-wiki: docs/maintain-guides-how-to-nominate-polkadot.md

---
id: maintain-guides-how-to-nominate-polkadot
title: Nomination Guide
sidebar_label: Nomination Guide
---

> The following information applies to the Polkadot network which is currently not yet launched. If
> you want to get started today please consider using this guide on the Westend test network, or
> checking out the [Kusama guide](maintain-guides-how-to-nominate-kusama) instead.

Nominators are one type of participant in the staking subsystem of Polkadot. They are responsible
for appointing their stake to the validators who are the second type of participant. By appointing
their stake, they are able to elect the active set of validators and share in the rewards that are
paid out.

> While the [validators][] are active participants in the network that take part in the block

Reviewer: Link to validator page

> production and finality mechanisms, nominators take a more passive role that can compared to the

Reviewer: ```suggestion
production and finality mechanisms, nominators take a more passive role with a
```

"set-it-and-forget-it" approach. Being a nominator does not require running a node of your own or

> worrying about online uptime. However, a good nominator performs diligence on the validators that it

Reviewer: ```suggestion
worrying about online uptime. However, a good nominator performs diligence on the validators that they
```

> elects. When looking for validators to nominate, a nominator should pay attention to its own reward

Reviewer: ```suggestion
elect. When looking for validators to nominate, a nominator should pay attention to their own reward
```

> percentage for nominating a specific validator - as well as the risk that it bares of being slashed

Reviewer: ```suggestion
percentage for nominating a specific validator - as well as the risk that they bare of being slashed
```

if the validator gets slashed.

# Setting up Stash and Controller keys

> Nominators are recommended to set up two separate stash and controller accounts. Explanation and

Reviewer: ```suggestion
Nominators are recommended to set up separate stash and controller accounts. Explanation and
```
The two here kind of sounds like they need two stash and two controller accounts

reasoning for generating distinct accounts for this purpose is elaborated in the [keys][] section of
the Wiki.

You can generate your stash and controller account via any of the recommended methods that are
detailed on the [account generation][] page.

# Using Polkadot UI

### Step 1: Bond your tokens

> 

Reviewer: Would maybe add a note here that this will be applicable when NPoS change happens. There's a disclaimer at the beginning, but also adding a brief one here wouldn't hurt.

stake, how much they charge in commission, the era points they've earned in the current era, and the
last block number that they produced. If you click on the chart button it will take you to the
"Validator Stats" page for that validator that shows you more detailed and historical information
about the validator's stake, rewards and slashes.

The "Waiting" subsection ([link](https://polkadot.js.org/apps/#/staking/waiting)) lists all pending
validators that are awaiting more nominations to enter the active validator set. Validators will
stay in the waiting queue until they have enough DOTs backing them (as allocated through the
Phragmen election mechanism). It is possible validator can remain in the queue for a very long time
if they never get enough backing.

The "Targets" subsection ([link](https://polkadot.js.org/apps/#/staking/targets)) will help you
estimate your earnings and this is where it's good to start picking favorites.

The "Account actions" subsection ([link](https://polkadot.js.org/apps/#/staking/actions)) allows you
to stake and nominate.

The "Validator Stats" screen will show you some interesting in-depth graphs about a selected

> validator.

Reviewer: Would copy the changes from above into here as well.

left over to pay transaction fees. Transaction fees are currently around 0.01 DOTs, but they are
dynamic based on a variety of factors including the load of recent blocks.

Also be mindful of the reaping threshold - the amount that must remain in an account lest it be
burned. That amount is 0.01 DOT on Polkadot, so it's recommended to keep around 0.1 DOT in your
account to be on the safe side.

Choose whatever payment destination that makes sense to you. If you're unsure, you can choose "Stash
account (increase amount at stake)" to simply accrue the rewards into the amount you're staking.

### Step 2: Nominate a validator

You are now bonded. Being bonded means your tokens are locked and could be
[slashed](learn-staking#slashing) if the validators you nominate misbehave. All bonded funds can now
be distributed to up to 16 validators. Be careful about the validators you choose since you will be
slashed if your validator commits an offence.

Click on "Nominate" on an account you've bonded and you will be presented with another popup asking
you to select up to 16 validators.

![Nominating validators](/img/NPoS/nominate.png)

> 

Reviewer: Would add a note here that although you can nominate all 16, the stake will be dispersed via phragmen, and you may not actually actively nominate all the ones you choose.

---
id: maintain-guides-how-to-nominate-polkadot
title: Nomination Guide
sidebar_label: Nomination Guide
---

> The following information applies to the Polkadot network which is currently in the soft launch
> phase. During soft launch the network starts as a Proof-of-Authority network before transitioning
> to Proof-of-Stake. You will be able to follow this guide to set up your validator but the first
> validator election and rewards will not start until later.

Nominators are one type of participant in the staking subsystem of Polkadot. They are responsible
for appointing their stake to the validators who are the second type of participant. By appointing
their stake, they are able to elect the active set of validators and share in the rewards that are
paid out.

While the [validators][] are active participants in the network that take part in the block
production and finality mechanisms, nominators take a more passive role with a
"set-it-and-forget-it" approach. Being a nominator does not require running a node of your own or

> worrying about online uptime. However, a good nominator performs diligence on the validators that

Reviewer: ```suggestion
worrying about online uptime. However, a good nominator performs due diligence on the validators that
```
